yoti-sdk-api/src/main/java/com/yoti/api/client/aml/AmlAddressFactory.java

```diff
@@ -0,0 +1,7 @@
+package com.yoti.api.client.aml;
+
+public interface AmlAddressFactory {
+
+    AmlAddress create(String postCode, String country);
+
```

Comment:
Do we need one with country only too?

Reply:
No, because there is a builder now.  They can do:
```
AmlAddress amlAddress = new AmlAddressBuilder()
      .withCountry("xyz")
      .build();
```